client/state/analytics/actions.js

```diff
@@ -81,3 +81,7 @@ export const recordPageView = ( url, title, service ) => ( {
 
 export const recordGooglePageView = ( url, title ) =>
 	recordPageView( url, title, 'ga' );
+
+export const startLuckyOrangeTracking = function( analytics ) {
+	analytics.luckyOrange.initialize();
+};
```

Comment:
The actual side effect -- the initialization of `luckyOrange` should go to `state/analytics/middleware` (for separation of concerns), similar to [these](https://github.com/Automattic/wp-calypso/blob/a7fab1ff6d3f7cbd3503d5d81d66152b2f2b722f/client/state/analytics/middleware.js#L11-L21).

Then that should be imported into this file, and used akin to [this stuff](https://github.com/Automattic/wp-calypso/blob/a7fab1ff6d3f7cbd3503d5d81d66152b2f2b722f/client/state/analytics/actions.js#L42-L83).

Since I believe that neither `recordEvent` nor `recordPageView` is a good fit for Lucky Orange -- given that what it does is best described as continuous tracking -- I'd suggest introducing a `startContinuousTracking` helper (and accompanying action name constants -- similar to what `recordEvent` or `recordPageView` do) and use that to implement `startLuckyOrangeTracking`.